about.html

```diff
@@ -15,25 +15,29 @@
 <body>
     <nav class="desktop-nav">
```

Comment:
-  Kindly, you forget to add the top short bar section with social media links, log out, and language links aligned to the right, please add it as it’s s part of the design 👇
- Kindly, give some padding for your logo and nav lists and give some space between your nav lists to match the given design also wrap your `nav` tag with `header` as it is part of the header for romantic reasons 👍 
### given design👇
![2023-03-25_08-36](https://user-images.githubusercontent.com/52010789/227698813-536fa215-6e27-433d-841f-7cbfd58e742e.png)
### your design 👇
![2023-03-25_08-39](https://user-images.githubusercontent.com/52010789/227698815-bc023cb6-3df0-4eb9-aa21-edbc98c4c0e4.png)